__test__/getProfiles.test.ts

```diff
@@ -0,0 +1,19 @@
+import { ThreadsAPI } from '../src/threads-api';
+
+test("UserName Unit test", async () => {
+  const threadsAPI = new ThreadsAPI();
+
+  const username = '_junhoyeo';
+  const id = await threadsAPI.getUserIDfromUsername(username);
+
+  if (!id) {
+    return;
+  }
+  expect(typeof id).toBe("string");
+
+  const user = await threadsAPI.getUserProfile(username, id);
+  expect(user.username).toBe(username);
+
+  const posts = await threadsAPI.getUserProfileThreads(username, id);
+  expect(Array.isArray(posts)).toBe(true)
+});
```

Comment:
Can you mind running `yarn format` before you commit your work?
If you're using VSCode, it should format automatically each time you save a file - if not, I must have messed up with the formatter settings... Will love if you could take a look!

Reply:
I see!